xonsh/completers/path.py

```diff
@@ -151,6 +151,59 @@ def _quote_paths(paths, start, end):
     return out
 
 
+def joinpath(path):
+    if path is None:
+        return ''
+    if len(path) == 0:
```

Comment:
Is there a reason these aren't `elif`?  Not that I imagine it makes too much difference, just curious.